Controllers/CommandsController.cs

```diff
@@ -0,0 +1,263 @@
+      .WithName("time-type")
+      .WithDescription(Locale.Get("eatcmd-timetype-description"))
+      .WithRequired(true)
+      .AddChoice(Locale.Get("seconds"), (long)1)
+      .AddChoice(Locale.Get("minutes"), (long)60)
+      .AddChoice(Locale.Get("hours"), (long)60 * 60)
+      .AddChoice(Locale.Get("days"), (long)60 * 60 * 24)
+      .WithType(ApplicationCommandOptionType.Integer)
+    );
+    eatCmdBuilder.AddOption("time-value", ApplicationCommandOptionType.Integer, Locale.Get("eatcmd-timevalue-description"), true);
+    eatCmdBuilder.AddOption("channel", ApplicationCommandOptionType.Channel, Locale.Get("eatcmd-channel-description"), false, null, false, null, null, null, channels);
+    eatCmdBuilder.AddOption("eat-existing", ApplicationCommandOptionType.Boolean, Locale.Get("eatcmd-eatexisting-description"), false);
+    eatCmdBuilder.AddOption("eat-future", ApplicationCommandOptionType.Boolean, Locale.Get("eatcmd-eatfuture-description"), false);
+
+    // Build donteat command
+    SlashCommandBuilder donteatCmdBuilder = new();
+    donteatCmdBuilder.WithName("donteat");
+    donteatCmdBuilder.WithDescription(Locale.Get("donteatcmd-description"));
+    donteatCmdBuilder.AddOption("channel", ApplicationCommandOptionType.Channel, Locale.Get("donteatcmd-channel-description"), false, null, false, null, null, null, channels);
+
+    // Build sniff command
+    SlashCommandBuilder sniffCmdBuilder = new();
+    sniffCmdBuilder.WithName("sniff");
+    sniffCmdBuilder.WithDescription(Locale.Get("sniffcmd-description"));
+
+    // Create commands
+    try {
+      await DiscordHelper.client.CreateGlobalApplicationCommandAsync(eatCmdBuilder.Build());
+      await DiscordHelper.client.CreateGlobalApplicationCommandAsync(donteatCmdBuilder.Build());
+      await DiscordHelper.client.CreateGlobalApplicationCommandAsync(sniffCmdBuilder.Build());
+    } catch (HttpException ex) {
+      string json = JsonConvert.SerializeObject(ex.Errors, Formatting.Indented);
+      Console.WriteLine(json);
+    }
+
+    Console.WriteLine("[CommandsController] Commands created successfully.");
+  }
+
+  public static async void EatHandler(SocketSlashCommand cmd) {
+    long timeType = 0;
+    long timeValue = 0;
+    IGuildChannel? channel = null;
+    bool eatExisting = false;
+    bool eatFuture = false;
+
+    // Loop through command options and assign values to variables
+    foreach (SocketSlashCommandDataOption option in cmd.Data.Options) {
+      switch (option.Name) {
+        case "time-type":
+          timeType = (long)option.Value;
+          break;
+        case "time-value":
+          timeValue = (long)option.Value;
+          break;
+        case "channel":
+          channel = (IGuildChannel)option.Value;
+          break;
+        case "eat-existing":
+          eatExisting = (bool)option.Value;
+          break;
+        case "eat-future":
+          eatFuture = (bool)option.Value;
+          break;
+      }
+    }
```

Comment:
What you think of this? 
Performance is O(5n) rather than O(n), but bit more concise and clear?
```suggestion

    T GetDataValue<T>(string key, T @default) => 
        (T) cmd.Data.Options.FirstOrDefault(o => o.Name == key)?.Value ?? @);

    var timeType = GetDataValue("time-type", 0l);
    var timeValue = GetDataValue("time-value", 0l);
    var channel = GetDataValue<IGuildChannel?>("channel", null);
    var eatExisting = GetDataValue("eat-existing", false);
    var eatFuture = GetDataValue("eat-future", false);
```

Reply:
Yup, love that !